recipes/treelite/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "treelite" %}
+{% set version = "0.92" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/treelite/archive/{{ version }}.tar.gz
+  sha256: 17961a4f4c5933630dfee5c7129180e9d69660f3efa05f87bc8b168081bf3fd7
+
+build:
+  number: 0
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - cmake >=3.14
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libprotobuf >=3.10
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - python
+    - pip >=19.0
+    - numpy
+    - scipy
+  run:
+    - python
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('scipy') }}
```

Comment:
Would just list `scipy` and drop it from `host`. I don't think there is a way to link against `scipy`, but I could be wrong.

Reply:
Moved to `run`.